config/settings.yml

```diff
@@ -0,0 +1,67 @@
+common:
+  store_assets_in_file_system: true
+  aws_access_key_id: <%= ENV['AWS_ACCESS_KEY_ID'] %>
+  aws_secret_access_key: <%= ENV['AWS_SECRET_ACCESS_KEY'] %>
+  host: lvh.me:3000
+  url_with_port: http://lvh.me:3000
+  default_from_email:
+  host_for_email_with_www: www.localhost:3000
+
+  mailer:
+    delivery_method: test
+    smtp_settings:
+      user_name:
+      password:
+      address: mailtrap.io
+      port: 2525
+      authentication: plain
+
+  devise:
+    secret_key:
+
+  honebadger_key: 4e69bd56
+
+test:
+  mailer:
+    delivery_method: :test
+
+development:
+  mailer:
+    delivery_method: :smtp
+
+staging:
+  store_assets_in_file_system: false
+  s3_bucket: <%= ENV['AWS_S3_BUCKET'] %>
+  mailer:
+    delivery_method: :smtp
+
+    smtp_settings:
+      user_name:
+      password:
+      address: mailtrap.io
+      port: 2525
+      authentication: plain
+
+  host:
+  url_with_port:
+  host_for_email_with_www:
+
+production:
+  store_assets_in_file_system: false
+  s3_bucket: <%= ENV['AWS_S3_BUCKET'] %>
+  bcc: <%= ENV['EMAIL_BCC'] %>
+  mailer:
+    delivery_method: :smtp
+
+    smtp_settings:
+      user_name: <%=ENV['SENDGRID_USERNAME']%>
+      password: <%=ENV['SENDGRID_PASSWORD']%>
+      domain: <%=ENV['SMTP_DOMAIN']%>
+      address: smtp.sendgrid.net
+      port: 587
+      authentication: plain
+      enable_starttls_auto: true
+
+  host: <%= ENV['HOST'] %>
+  url_with_port: <%= ENV['HOST'] %>
+  host_for_email_with_www: <%= ENV['HOST'] %>
```

Comment:
@ershad We have moved Settings.yml to Rails application secrets, can you move/merge there?